lib/index.js

```diff
@@ -126,6 +127,37 @@ module.exports = {
         const SilentError = require('silent-error');
         throw new SilentError(message.join('\n\t'));
       }
+    } else {
+      this.ui.writeWarnLine(
+        'The Ember Classic edition has been deprecated. Speciying "classic" in your package.json, or not specifying a value at all, will no longer be supported. You must explicitly set the "ember.edition" property to "octane". This warning will become an error in Ember 4.0.0.\n\nFor more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_editions-classic'
+      );
+
+      if (
+        optionalFeaturesMissing ||
+        optionalFeatures.isFeatureEnabled('application-template-wrapper')
+      ) {
+        this.ui.writeWarnLine(
+          'Setting the `application-template-wrapper` optional feature flag to `true`, or not providing a setting at all, has been deprecated. You must add the `@ember/optional-features` addon and set this feature to `false`. You can also run `npx @ember/octanify` to do this. This warning will become an error in Ember 4.0.0.\n\nFor more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_optional-feature-application-template-wrapper'
+        );
+      }
+
+      if (
+        optionalFeaturesMissing ||
+        !optionalFeatures.isFeatureEnabled('template-only-glimmer-compoents')
+      ) {
+        this.ui.writeWarnLine(
+          'Setting the `template-only-glimmer-components` optional feature flag to `false`, or not providing a setting at all, has been deprecated. You must add the `@ember/optional-features` addon and set this feature to `true`. You can also run `npx @ember/octanify` to do this. This warning will become an error in Ember 4.0.0.\n\nFor more information, see the deprecation guide: https://deprecations.emberjs.com/v3.x/#toc_optional-feature-template-only-glimmer-components'
+        );
+      }
+    }
+
+    if (
+      optionalFeaturesMissing ||
+      optionalFeatures.isFeatureEnabled('optional-feature.jquery-integration')
```

Comment:
Is this right? I would expect it to be:

```suggestion
      optionalFeatures.isFeatureEnabled('jquery-integration')
```